manage-data/data-store/data-streams/failure-store.md

```diff
@@ -0,0 +1,611 @@
+
+```console
+POST my-datastream::failures/_rollover
+```
+
+```console-result
+{
+  "acknowledged": true,
+  "shards_acknowledged": true,
+  "old_index": ".fs-my-datastream-2025.05.01-000002",
+  "new_index": ".fs-my-datastream-2025.05.01-000003",
+  "rolled_over": true,
+  "dry_run": false,
+  "lazy": false,
+  "conditions": {}
+}
+```
+
+### Failure store lifecycle [manage-failure-store-lifecycle]
+
+Failure stores have their retention managed using an internal [data stream lifecycle](./failure-store.md). A thirty day (30d) retention is applied to failure store data. You can view the active lifecycle for a failure store index by calling the [get data stream API](./failure-store.md):
+
+```console
+GET _data_stream/my-datastream
+```
+
+```console-result
+{
+  "data_streams": [
+    {
+      "name": "my-datastream",
+      "timestamp_field": {
+        "name": "@timestamp"
+      },
+      "indices": [
+        {
+          "index_name": ".ds-my-datastream-2025.05.01-000001",
+          "index_uuid": "jUbUNf-8Re-Nca8vJkHnkA",
+          "managed_by": "Data stream lifecycle",
+          "prefer_ilm": true,
+          "index_mode": "standard"
+        }
+      ],
+      "generation": 2,
+      "status": "GREEN",
+      "template": "my-datastream-template",
+      "lifecycle": {
+        "enabled": true
+      },
+      "next_generation_managed_by": "Data stream lifecycle",
+      "prefer_ilm": true,
+      "hidden": false,
+      "system": false,
+      "allow_custom_routing": false,
+      "replicated": false,
+      "rollover_on_write": false,
+      "index_mode": "standard",
+      "failure_store": { <1>
+        "enabled": true,
+        "rollover_on_write": false,
+        "indices": [
+          {
+            "index_name": ".fs-my-datastream-2025.05.05-000002",
+            "index_uuid": "oYS2WsjkSKmdazWuS4RP9Q",
+            "managed_by": "Data stream lifecycle"  <2>
+          }
+        ],
+        "lifecycle": {
+          "enabled": true,
+          "effective_retention": "30d",  <3> 
+          "retention_determined_by": "default_failures_retention"  <4>
+        }
+      }
+    }
+  ]
+}
+```
+1. Information about the failure store is presented in the response under its own field.
+2. Indices are managed by data stream lifecycles by default.
+3. An effective retention period of thirty days (30d) is present by default.
+4. The retention is currently determined by the default.  
+
+:::{note}
+The default retention respects any maximum retention values. If [maximum retention](./failure-store.md) is configured lower than thirty days then the maximum retention will be used as the default value.
+:::
+
+You can update the default retention period for failure stores in your deployment by updating the `data_streams.lifecycle.retention.failures_default` cluster setting. Data streams that have no retention configured on their failure stores will use this value to determine their retention period.
```

Comment:
Will this change the retention for existing data streams?

Reply:
Yes, changed to "New and existing data streams that have no retention configured on their failure store will use this value..." in cec305f9